app/templates/bower.json

```diff
@@ -12,7 +12,6 @@
     "angular-x2js": "https://github.com/janmichaelyu/angular-x2js.git",
     "bootstrap": "~3.3.5",
     "font-awesome": "~4.4.0",
-    "highlightjs":"~8.7.0",
```

Comment:
This looks a bit suspicious, why delete this dependency? I think it is necessary. Run `bower prune` to make sure you are not running against dependencies that are no longer in bower.json..